docs/src/main/paradox/index.md

```diff
@@ -19,7 +23,15 @@ Based on a protobuf service definition, akka-grpc can generate:
 * On the server side, code to create an Akka HTTP route based on your implementation of the API
 * On the client side, a client for the API.
 
+### gRPC vs REST
+
+* Where REST is more flexible about transport and encoding, gRPC standardizes on HTTP/2 and Protobuf.
+* Where REST can be either schemaless or use a 3rd-party schema, gRPC always declares the service and messages in a Protobuf schema definition.
+
+### gRPC vs SOAP
 
+* Where SOAP is more flexible about transport and encoding, gRPC standardizes on HTTP/2 and Protobuf.
```

Comment:
is it more flexible really?

Reply:
Not about encoding I guess, transport kinda (but I guess that might not be widely used https://en.wikipedia.org/wiki/SOAP#Transport_methods )